docs/hubble/tutorials.md

```diff
@@ -3,4 +3,6 @@
 Guides to set up a Hubble instance using various cloud providers.
 
 - [AWS EC2](https://warpcast.notion.site/Set-up-Hubble-on-EC2-Public-23b4e81d8f604ca9bf8b68f4bb086042)
-- [Digital Ocean](https://warpcast.notion.site/Set-up-Hubble-on-DigitalOcean-Public-e38173c487874c91828665e73eac94c1)
+- [Digital Ocean](https://warpcast.notion.site/Set-up-Hubble-on-DigitalOcean-Public-e38173c487874c91828)
```

Comment:
Why was this Digital Ocean link changed? It's now broken—please restore.

Reply:
@sds I'am so sorry! The original Digital Ocean link has been restored as requested. Now is OK! Thanks, please review!

Reply:
@sds Could you please help me merge my issue and pull request number #257?